packages/experiments/src/components/Chiclet/ChicletXsmall.base.tsx

```diff
@@ -23,7 +26,14 @@ export class ChicletXsmallBase extends React.Component<IChicletCardProps, {}> {
 
     return (
       <div tabIndex={tabIndex} role={role} onClick={this._onClick} className={this._classNames.root}>
-        {generateCommonHTML(this.props, imageProvided, this._classNames.icon, this._classNames.preview)}
+        {generateCommonHTML(
```

Comment:
This function starts to have too many arguments passed in. In a future iteration you might want to aggregate them in some sort of object.